app/commands/git/sync_doc.rb

```diff
@@ -15,16 +16,15 @@ def call
       ) { |d| d.attributes = attributes }
 
       doc.update!(attributes)
-    rescue StandardError
-      # TODO: (Blocking): Raise issue on GH.
+    rescue StandardError => e
+      Github::Issue::OpenForDocSyncFailure.(config, section, e, git_sha)
     end
 
     private
-    attr_reader :config, :section, :track
+    attr_reader :config, :section, :track, :git_sha
 
     def repo_url
-      # TODO: (Optional): Put a constant somewhere for this
-      track ? track.repo_url : "https://github.com/exercism/docs"
+      track ? track.repo_url : Document::REPO_URL
```

Comment:
I've moved the repo URL into a constant on the `Document` model. Let me know if you think that's an ok place.

Reply:
Yep - that's good!